wgpu/backends/wgpu_native/extras.py

```diff
@@ -33,3 +33,32 @@ def request_device_tracing(
     return adapter._request_device(
         label, required_features, required_limits, default_queue, trace_path
     )
+
+
+def create_pipeline_layout(
```

Comment:
Quick question for @almarklein.  I noticed that for request_device, you gave the version in extras.py the name request_device_tracing.

I made the extras.py version of create_pipeline_layout have the same name.  I prefer that, but I'll change the name if you want consistency.

Reply:
Yeah good point. In that case it was pretty obvious, because it only added the tracing feature. But this won't hold up when the wgpu_native version of a function supports multiple extra functionalities. I like what you did. I think it would be good to rename `request_device_tracing` in a new pr after this.